README.md

```diff
@@ -1 +1,9 @@
-# product-landing-page-trails
```

Comment:
The readme is not professional,why don't you add content,listing the technology used in the project and how to run it ,because it shows your readme looks more professional and visitors to project repo will have better understanding